index.html

```diff
@@ -67,117 +67,82 @@ <h1>Learning that gets you</h1>
-                <li class="course-bar-item">Data Science</li>
-                <li class="course-bar-item">AWS Certification</li>
-                <li class="course-bar-item">Drawing</li>
+            <ul class="course-bar mb-3">
+                <li class="course-bar-item course-bar-item-active" href="http://localhost:3000/Python">Python</li>
+                <li class="course-bar-item" href="http://localhost:3000/Excel">Excel</li>
+                <li class="course-bar-item" href="http://localhost:3000/Web">Web Development</li>
+                <li class="course-bar-item" href="http://localhost:3000/JavaScript">JavaScript</li>
+                <li class="course-bar-item" href="http://localhost:3000/DataScience">Data Science</li>
+                <li class="course-bar-item" href="http://localhost:3000/AWS">AWS Certification</li>
+                <li class="course-bar-item" href="http://localhost:3000/Drawing">Drawing</li>
             </ul>
-            <div class="course-container">
-                <h3 class="course-item">Expand your career opportunities with Python</h3>
-                <p class="course-item paragraph">
-                    Take one of Udemy’s range of Python courses and learn how to code using this incredibly useful language. Its simple syntax and readability makes Python perfect for Flask, Django, data science, and machine learning. You’ll learn how to build everything
-                    from games to sites to apps. Choose from a range of courses that will appeal to ...
-                </p>
-                <button class="course-item btn-outline">Explore Python</button>
-                <div class="courses-container">
-                    <div class="course">
-                        <img src="images/python1.jpg" alt="python course photo" class="course-detail">
-                        <h3 class="course-title course-detail">Learn Python: The Complete Python Programming Course</h3>
-                        <p class="course-author course-detail">Avinash Jain, The Codex</p>
-                        <div class="course-detail">
-                            <span class="rating ">4.4</span>
-                            <span class="rating ">
-                            <i class="fa-solid fa-star"></i>
-                            <i class="fa-solid fa-star"></i>
-                            <i class="fa-solid fa-star"></i>
-                            <i class="fa-solid fa-star"></i>
-                            <i class="fa-regular fa-star-half-stroke"></i>
-                            </span>
-                            <span class="rating-numbers course-author">(2,906)</span>
+            <div class="course-container ">
+
+            </div>
+        </section>
+        <section class="category-section">
+            <h2 class="mt-1">Top categories</h2>
+            <div class="category-content">
+                <div class="row">
+                    <div class="col mb-1">
```

Comment:
in the [requirements](https://www.notion.so/Project1-phase3-8d972b7bc7224da293ebe50689845832)
we need it display 4 courses in large screen, 3 in medium, 1 in small

Reply:
okay will solve it